src/java/org/apache/cassandra/streaming/StreamSession.java

```diff
@@ -535,8 +536,8 @@ private void abortTasks()
      */
     public void state(State newState)
     {
-        if (logger.isTraceEnabled())
-            logger.trace("[Stream #{}] Changing session state from {} to {}", planId(), state, newState);
+        if (logger.isDebugEnabled())
+            logger.debug("[Stream #{}] Changing session state from {} to {}", planId(), state, newState, new Throwable("caller"));
```

Comment:
the throwable was just for debugging to see who set the state, will remove before merging